sns_ik_lib/CMakeLists.txt

```diff
@@ -33,11 +33,6 @@ catkin_package(
 include_directories(include utilities ${catkin_INCLUDE_DIRS} ${EIGEN3_INCLUDE_DIRS} ${orocos_kdl_INCLUDE_DIRS})
 link_directories(${orocos_kdl_LIBRARY_DIRS})
 
-# library for private utilities
-add_library(sns_ik_util
-            utilities/sns_ik_math_utils.cpp
-            utilities/sns_linear_solver.cpp)
-
 # library for public API
```

Comment:
This comment is somewhat misleading - can you update it?

Reply:
This is the library for the public API, so the comment seems fine to me. Can you elaborate on what is misleading? The API itself is the header files that are installed.

Reply:
Fair point. I think that my confusion was mostly about having the utilities listed under the public API comment, even though they are not technically part of the header files that are installed. It is fine though, because these utilities are technically part of the same library as all of the header files that are exported.